discord-scripts/fjord.ts

```diff
@@ -7,74 +7,115 @@ export default async function manageFjord(
   discordClient: Client,
   robot: Robot<any>,
 ) {
+  const guildId = "597157463033100784"
+  const guild = discordClient.guilds.cache.get(guildId)
+
+  if (!guild) {
+    robot.logger.error(`Guild with ID ${guildId} not found.`)
+    return
+  }
+
+  guild.commands.set([
```

Comment:
I did some testing with this and the good news is it seems that since this is a guild based command set, it will **not** overwrite the global commands set to the application (i.e Figma integration). So perhaps best moving forward, refactor all commands into a `global-commands.ts` file we can setup in `discord-scripts` for deploying commands to the application. Curious to hear your thoughts!

Reply:
I would prefer for scripts to self-manage commands tbh. A global list is just another thing to maintain, and means we can't easily share parts of our bot with spun out projects.